lib/YaST/DNSServer/Sle/DNSServerController.pm

```diff
@@ -0,0 +1,25 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: The class introduces actions for DNS Server Settings Page.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package YaST::DNSServer::Sle::DNSServerController;
+use parent 'YaST::DNSServer::DNSServerController';
+use strict;
+use warnings;
+
+sub init {
```

Comment:
we don't need to overwrite `init` if we don't change anything.

Reply:
updated.